nuxeo-web-ui-ftest/wdio.conf.js

```diff
@@ -1,5 +1,59 @@
 const path = require('path');
 
+const reporters = ['spec'];
+if (process.env.JUNIT_REPORT_PATH) {
+  reporters.push('junit');
+}
+if (process.env.CUCUMBER_REPORT_PATH) {
+  reporters.push('multiple-cucumber-html');
+}
+
+const plugins = {};
+plugins[path.join(__dirname, 'wdio-shadow-plugin')] = {};
+
+const capability = {
+  // maxInstances can get overwritten per capability. So if you have an in-house Selenium
+  // grid with only 5 firefox instance available you can make sure that not more than
+  // 5 instance gets started at a time.
+  maxInstances: 1,
+  browserName: process.env.BROWSER,
+  javascriptEnabled: true,
+  acceptSslCerts: true,
+};
+
+switch (capability.browserName) {
+  case 'chrome':
+    capability.chromeOptions = {
+      args: [
+        '--no-sandbox',
+        // '--auto-open-devtools-for-tabs',
+        // '--window-size=1920,1080',
+        // '--headless',
+      ],
+    };
+    if (process.env.BROWSER_BINARY) {
+      capability.chromeOptions.binary = process.env.BROWSER_BINARY;
+    }
+    break;
+  case 'firefox':
+    capability['moz:firefoxOptions'] = {
+      args: [
+        // '-headless',
+      ],
+    };
+    if (process.env.BROWSER_BINARY) {
+      capability['moz:firefoxOptions'].binary = process.env.BROWSER_BINARY;
+    }
+    break;
+  case 'safari':
+    capability['safari.options'] = {
+      technologyPreview: false,
+    };
+    break;
+  default:
+    throw new Error('Invalid browser');
```

Comment:
This will prevent tests from running on Edge. If we want to have a default, we must add a `case 'edge':` statement.

Reply:
Thx 👍 Good catch, did not mean to introduce default, guess I messed up one of the 1000 rebases ;)